packages/http/src/router/__tests__/utils.ts

```diff
@@ -9,11 +9,11 @@ export function pickOneHttpMethod(): HttpMethod {
   return chance.pickone(httpMethods);
 }
 
-export function pickSetOfHttpMethods(count: number = 2): HttpMethod[] {
+export function pickSetOfHttpMethods(count = 2): HttpMethod[] {
```

Comment:
hm.. so this is in the spirit of `prefer inference over explicitness` kinda thing?

Reply:
Apparently so — I'm kind of ok here, it's a number and it's pretty clear it's a number.